redbot/core/utils/views.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+
+import discord
+
+from typing import TYPE_CHECKING, List, Union, Dict
+from redbot.core.i18n import Translator
+from redbot.vendored.discord.ext import menus
+
+
+if TYPE_CHECKING:
+    from redbot.core.commands import Context
+
+_ = Translator("UtilsViews", __file__)
+
+
+class _SimplePageSource(menus.ListPageSource):
+    def __init__(self, items: List[Union[str, discord.Embed]]):
+        super().__init__(items, per_page=1)
+
+    async def format_page(
+        self, menu: menus.MenuPages, page: Union[str, discord.Embed]
+    ) -> Union[str, discord.Embed]:
+        return page
+
+
+class _NavigateButton(discord.ui.Button):
+    def __init__(
+        self, style: discord.ButtonStyle, emoji: Union[str, discord.PartialEmoji], direction: int
+    ):
+        super().__init__(style=style, emoji=emoji)
+        self.direction = direction
+
+    async def callback(self, interaction: discord.Interaction):
+        if self.direction == 0:
+            self.view.current_page = 0
+        elif self.direction == self.view.source.get_max_pages():
+            self.view.current_page = self.view.source.get_max_pages() - 1
+        else:
+            self.view.current_page += self.direction
+        kwargs = await self.view.get_page(self.view.current_page)
+        await interaction.response.edit_message(**kwargs)
+
+
+class SimpleMenu(discord.ui.View):
```

Comment:
I think Views should be suffixed with View (`SimpleMenuView`? `PaginatedView`?) so it isn't confused with menus from the old ext library.

Reply:
I don't see any confusion here? SimpleMenu could be put into the utils.menus namespace but since it's a view I think it makes more sense inside the new utils.views namespace.

Reply:
I think SimpleMenu being placed here in views is totally fine and appropriate for its function and goal.